neuraxle/steps/flow.py

```diff
@@ -23,61 +23,18 @@
     project, visit https://www.umaneo.com/ for more information on Umaneo Technologies Inc.
 
 """
-from abc import abstractmethod
 from typing import Union
 
-from neuraxle.base import BaseStep, MetaStepMixin, DataContainer, ExecutionContext, TruncableSteps, ResumableStepMixin
+from neuraxle.base import BaseStep, MetaStepMixin, DataContainer, ExecutionContext, TruncableSteps, ResumableStepMixin, \
+    HandlerMixin, NonFittableMixin, TransformHandlerMixin
 from neuraxle.data_container import ExpandedDataContainer
 from neuraxle.hyperparams.space import HyperparameterSamples
 from neuraxle.union import FeatureUnion
 
-
-class ForceMustHandleMixin:
```

Comment:
I really like this old name. I would revert to that. It seems dead-clear, or almost, that here it forces handling and that it fails on regular calls.